storybook/storybook-docs/src/interactive-elements.stories.mdx

```diff
@@ -0,0 +1,66 @@
+
+### Relevante WCAG eisen:
+
+- [WCAG 1.4.1](https://www.w3.org/TR/WCAG22/#use-of-color): kleur mag niet de enige manier zijn waarop je aangeeft dat iets interactief is
+- [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum): tekst heeft voldoende contrast met de achtergrond
+
+## Formaat
+
+[Interactieve elementen moeten minstens 24 bij 24 pixels groot zijn.](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)
+
+## Focus indicator
+
+Wij gebruiken grotendeels de standaard focus indicator van de webbrowsers. De enige aanpassing die we doen is de focus `outline` een `offset` geven van 2 pixels, zodat er ruimte zit tussen het element en de focus indicator. Hierdoor heeft de focus indicator ook voldoende contrast met donkere elementen.
+
+Let op bij het inrichten van een pagina dat de interactieve elementen in een logische volgorde staan, meestal van boven naar beneden en van links naar rechts. Hierdoor krijg je een logische focus / tabbing order.
+
+### Relevante WCAG eisen
+
+- [WCAG 2.4.3](https://www.w3.org/TR/WCAG22/#focus-order): interactieve elementen hebben een logische focus / tabbing order
+- [WCAG 2.4.7](https://www.w3.org/TR/WCAG22/#focus-visible): de focus indicator is zichtbaar
+- [WCAG 2.4.11](https://www.w3.org/TR/WCAG22/#focus-not-obscured-minimum): de focus indicator wordt niet volledig verborgen door content
+- [WCAG 2.4.12](https://www.w3.org/TR/WCAG22/#focus-not-obscured-enhanced): de focus indicator wordt niet deels verborgen door content
+- [WCAG 2.4.13](https://www.w3.org/TR/WCAG22/#focus-appearance): de focus indicator is minimaal 2 pixels breed en heeft voldoende contrast met het element dat focus heeft
+
+## Hover
+
+Gebruik verschillende, niet overlappende stijlen voor hover en focus. Als toetsenbordgebruiker is het belangrijk dat je weet welk element je activeert. Als de hover-variant (bijna) hetzelfde is als de focus-variant, dan weet je niet zeker welk element je zult activeren.
+
+We gebruiken de cursor hand `pointer` bij hover op alle interactieve elementen die geen eigen cursor stijl hebben (zoals tekstvakken). Oorspronkelijk was `pointer` alleen bedoeld voor links, maar wij vinden dat de associatie van `pointer` met interactiviteit zo sterk is dat we deze cursor vaker gebruiken.
+
+Gebruik het `title` attribuut niet. Interactieve elementen met het `title` attribuut kunnen een tooltip tonen die over de tekst heen staat, waardoor het minder leesbaar is.
+
+### Referenties
+
+- [Documentatie van Utrecht over focus en hover](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-link--docs)
+- [Specificatie van cursor pointer.](https://www.w3.org/TR/css-ui-4/#valdef-cursor-pointer) Hierin staat dat je pointer moet gebruiken voor links, en mag gebruiken voor andere interactieve elementen
+- [Artikel over het niet gebruiken van cursor pointer voor andere elementen dan links.](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b)
```

Comment:
This contradicts your statement about using pointer
> Oorspronkelijk was `pointer` alleen bedoeld voor links, maar wij vinden dat de associatie van `pointer` met interactiviteit zo sterk is dat we deze cursor vaker gebruiken.

Reply:
I added this to show both sides to the argument, do you think it isn't necessary?

Reply:
I agree it may be confusing if left in without context. I can live with removing it.